package.json

```diff
@@ -4,6 +4,11 @@
   "engines": {
     "node": ">=0.10"
   },
+  "files": [
+    "src",
+    "lib",
```

Comment:
what is lib?

Reply:
It's the output dir of TypeScript, where compiled JavaScript goes.

https://github.com/opentracing/opentracing-javascript/blob/062505c9327c964c2925caaed9aa994ce5684f5e/tsconfig.json#L11

Reply:
how is that different from dist?

Reply:
`src` has `.ts` files. `dist` has `.js`, `.js.map`, `.d.ts` and `.d.ts.map` files (result of compiling `.ts` fields in `src`).
Note that that the only thing new here is generation of the `.d.ts.map` files and inclusion of `src` in the npm package.

You can compare https://unpkg.com/opentracing@0.14.3/dist/ to https://github.com/opentracing/opentracing-javascript/tree/master/src if you're curious about the differences of the files.